app/controllers/projects_controller.rb

```diff
@@ -3,8 +3,11 @@
 class ProjectsController < ApplicationController
   def index
     @projects = Project.unarchived.by_last_updated.page(params[:page]).per(7)
-    @entry = Entry.new
-    @activities = Entry.by_last_created_at.limit(30)
+    @hours_entry = Hour.new
+    @mileages_entry = Mileage.new
+    @activities = Hour.by_last_created_at.limit(30)
+    @mileages = mileages?
```

Comment:
this instance variable is used in the projects view to determine what form to render (hours or mileages)